runtime/onert/sample/minimal-python/README.md

```diff
@@ -0,0 +1,146 @@
+# minimal-python
+
+`minimal-python` is a simple driver to run `nnpackage` with nnfw python API.
+
+It takes `nnpackage` as input. It uses **nnfw python API** internally.
+
+It assumes model of float32 tensor type as an input.
+
+## Usage
+
+```
+$ python3 minimal.py path_to_nnpackage_directory
+```
+
+## Module Description
```

Comment:
I think Module Description can be removed. It does not give more information than source code.

Reply:
I'll remove Module Description ASAP.
Then Is there no need to write Module Description in other space?

Reply:
If `Module Description` does not add more value (i.e. it is not hard to find the same info from source), it can be eliminated. It is likely to be out-of-dated as code changes.